blog-post.md

```diff
@@ -139,4 +139,4 @@ As mentioned, Lit helps smooth out some of the things about Web Components that
 
 ## How We Use Lit
 
-Here at GitKraken, we use Lit to share components across our entire platform. The most prominent example is the Commit Graph. This allows us to share functionality between our Desktop client and GitLens in VSCode. As we bring the Commit Graph into other projects, we will be able to keep the experience clean and consistent from a styling and functionality perspective. 
+Here at GitKraken, we use Lit to share components across our entire platform. The most prominent example is the Commit Graph. This allows us to share functionality between our Desktop client and GitLens in VSCode. As we bring the Commit Graph into other projects, we will be able to keep the experience clean and consistent from a styling and functionality perspective.   
```

Comment:
> The most prominent example is the Commit Graph. This allows us to share functionality between our Desktop client and GitLens in VSCode. As we bring the Commit Graph into other projects, we will be able to keep the experience clean and consistent from a styling and functionality perspective.

To my knowledge, the Commit Graph is a collection of shared React components and not shared web components.

I also find it tricky to describe how we are using Web Components across our products because, quite frankly, we are using them inconsistently and not fully to consolidate/standardize styles. 

It may be more accurate to describe how GitLens utilizes web components in order to reduce their reliance on React. But, in all transparency, Web Components have not been widely integrated into other surfaces.

Reply:
Great feedback. Thanks. Gyo also mentioned that the Commit Graph is NOT a set of Lit components. So I will make that revision.

As per our Discord discussion, I will adjust the "How we use it" section to call out the components we DO use.